CommonClient.py

```diff
@@ -108,46 +122,71 @@ def _cmd_missing(self, filter_text = "") -> bool:
 
         if count:
             self.output(
-                f"Found {count} missing location checks{f'. {checked_count} location checks previously visited.' if checked_count else ''}")
+                f"Found {count} missing location checks{f" matching '{filter_text}'" if filter_text else ""}{f'. {checked_count} location checks previously visited.' if checked_count else ''}")
         else:
-            self.output("No missing location checks found.")
+            self.output(f"No missing location checks{f" matching '{filter_text}'" if filter_text else ""} found.")
```

Comment:
nesting f-strings like this (repeating the same kind of quotes) doesn't work in python 3.8 (nor 3.11)

Reply:
ah, of course there's a problem...